interp/api.go

```diff
@@ -313,6 +319,14 @@ func OpenHandler(f OpenHandlerFunc) RunnerOption {
 	}
 }
 
+// ReadDirHandler sets the read directory handler. See ReadDirHandlerFunc for more info.
+func ReadDirHandler(f ReadDirHandlerFunc) RunnerOption {
```

Comment:
you remind me that I regret going for functional options :) filed https://github.com/mvdan/sh/issues/801.